src/test/java/sky/command/DeadlineCommandTest.java

```diff
@@ -0,0 +1,43 @@
+package sky.command;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import java.io.IOException;
+
+import org.junit.jupiter.api.Test;
+
+import sky.Sky;
+import sky.TaskList;
+import sky.exception.TextNoMeaningException;
+import sky.storage.History;
+import sky.storage.Storage;
+
+
+public class DeadlineCommandTest {
+    private Sky sky;
+    private TaskList taskList = new TaskList();
+    private Storage storage = new Storage("data/sky.txt");
+    private History history = new History(this.sky, this.taskList);
+
+    @Test
+    public void dummyTest() {
+        assertEquals(2, 2);
+    }
+
+    @Test
+    public void deadlineTestOnEmptyList() {
```

Comment:
I noticed that your test methods do not follow the recommended format of `featureUnderTest_testScenario_expectedBehavior()`. Perhaps you can consider changing your test methods to this naming convention for easier reference? Otherwise, I think it should be fine to leave them as they are as well 😊